src/main/java/me/totalfreedom/totalfreedommod/command/Command_gcmd.java

```diff
@@ -11,46 +14,73 @@
 public class Command_gcmd extends FreedomCommand
 {
 
+    public static final List<String> BLOCKED_COMMANDS = Arrays.asList(
```

Comment:
As a list in the config is how I would do it, then run a check to see if the command is in that list when gcmd is ran.